src/bosh-softlayer-cpi/softlayer/common/agent_options.go

```diff
@@ -42,6 +42,14 @@ func (o AgentOptions) Validate() error {
 	return nil
 }
 
+func (o *AgentOptions) GetVcapPassword() string {
+	return o.vcapPassword
+}
+
+func (o *AgentOptions) SetVcapPassword(vcapPassword string) {
```

Comment:
@maximilien I reviewed the code, vcap password has to be used in other packages, like [softlayer_virtual_guest_creator.go](https://github.com/cloudfoundry/bosh-softlayer-cpi-release/blob/master/src/bosh-softlayer-cpi/softlayer/vm/softlayer_virtual_guest_creator.go#L146). But please note that the password is encrypted with SHA-512 (when it's set in cpi.json). Do you think if it's still a case? Thanks.